examples/basic-jsx-precompile/index.html

```diff
@@ -17,9 +17,17 @@
     </div>
     <h4>Example Details</h4>
     <p>This is written with JSX in a separate file and precompiled to vanilla JS by running:</p>
+
+    <p><i>With Babel lower than 6.0</i></p>
     <pre>npm install -g babel
 cd examples/basic-jsx-precompile/
 babel example.js --out-dir=build</pre>
+
+	<p><i>With Babel 6.0 or higher</i></p>
+    <pre>npm install -g babel-cli
+npm install babel-preset-react
+cd examples/basic-jsx-precompile/
+babel example.js --presets react --out-dir=build</pre>
```

Comment:
According to @zpao 's comment, I removed the instruction to create a file and added the --presets flag. I also optimized line wrapping for github.